### PR TITLE
Clean up APIAP traces related to integrations show

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-7068: Remove Integrations page related code](https://issues.redhat.com/browse/THREESCALE-7068)

### CODE REVIEW

**Review by hallelujah — APPROVED** [review]

File: app/helpers/api/integrations_helper.rb [hunk 1]
```diff
@@ -69,11 +65,6 @@ def deployment_option_is_service_mesh?(service)
     service.deployment_option =~ /^service_mesh/
   end
 
-  def edit_deployment_option_title(service)
```
Comment (guicassolato) [inline app/helpers/api/integrations_helper.rb:72]: Can this like finally be removed as well https://github.com/3scale/porta/blob/80daaa207a1974888083a2bab96a2ed6168290b0/app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim#L3?

Comment (josemigallas) [reply app/helpers/api/integrations_helper.rb:72]: I missed that one, thanks!

File: app/views/api/integrations/apicast/shared/_deployment_options.html.slim [hunk 2]
```diff
@@ -1,33 +0,0 @@
-.SettingsBox
```
[the remark this answers is not in the document]
Comment (josemigallas) [reply app/views/api/integrations/apicast/shared/_deployment_options.html.slim:1]: Rebased

**Review by guicassolato — APPROVED** [review]
It would be nice to have this tested in preview just in case. But it LGTM overall.

**Comment by josemigallas** [comment]
> It would be nice to have this tested in preview just in case. But it LGTM overall.

I'll deploy it and test it before merging. Algo ignoring the cucumber job failure as it comes from master.

**Pull request merged**